environment_cpu.yaml

```diff
@@ -1,22 +1,20 @@
 name: sam
 channels:
-    - pytorch
     - conda-forge
 dependencies:
-    - cpuonly
-    # This pin is necessary because later nifty versions have import errors on windows.
-    - nifty =1.2.1=*_4
+    - nifty >=1.2.1
     - imagecodecs
     - magicgui
     - napari >=0.5.0
     - natsort
     - pip
     - pooch
-    - protobuf <5
     - pyqt
     - python-xxhash
     - python-elf >=0.4.8
-    - pytorch >=2.4
+    # This does not work for me.
+    # - "pytorch=*=cpu"
+    - pytorch-cpu
```

Comment:
```suggestion
    - "pytorch=*=cpu*"
    # - pytorch-cpu
```
you need the * on the cpu to catch the rest of the build number
![image](https://github.com/user-attachments/assets/e6731f2b-01b3-4ae0-a91b-8f14054ba663)